velox/exec/AddressableNonNullValueList.cpp

```diff
@@ -18,24 +18,29 @@
 
 namespace facebook::velox::aggregate::prestosql {
 
-AddressableNonNullValueList::Entry AddressableNonNullValueList::append(
-    const DecodedVector& decoded,
-    vector_size_t index,
+void AddressableNonNullValueList::initStream(
```

Comment:
perhaps, return ByteOutputStream

Reply:
I see that ByteOutputStream has a deleted copy-constructor. This prevents copying and moving. But you can define move constructor to enable std::move.

https://en.cppreference.com/w/cpp/language/move_constructor

Reply:
Fixed the code.